### PR TITLE
[New Model]: Failure Pattern

### PR DESCRIPTION
**Pull request by JulianSchererZF**
## Description

 -->

Closes #[620](https://github.com/eclipse-tractusx/sldt-semantic-models/issues/620)

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by JulianSchererZF** [comment]
@agg3fe ready for review

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 1]
```diff
@@ -0,0 +1,272 @@
+
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:1]: copyright headers are missing

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+   samm:description "This characteristic groups all necessary properties to refer to an existing Catena-X aspect model."@en ;
+   samm:dataType :AspectModel .
+
+:BooleanInsideOperatorEnumerator a samm-c:Enumeration ;
+   samm:preferredName "Boolean inside operator enumeration"@en ;
+   samm:description "Provides a selection of several logical operators for boolean expressions"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "equal" "not_equal" ) .
+
+:TimtstampInsideOperatorEnumeration a samm-c:Enumeration ;
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:202]: typo in `TimtstampInsideOperatorEnumeration`

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+   samm:description "This regular expression ensures that the version is [1-9].[0-9].[0.9]"@en ;
+   samm:value "^[1-9].[0-9].[0-9]$" .
+
+:StringExpressionList a samm-c:List ;
+   samm:dataType :StringExpression .
+
+:DateExpressionList a samm-c:List ;
+   samm:dataType :TimestampExpression .
+
+:BooleanExpressionList a samm-c:List ;
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:266]: preferredName and description missing

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+   samm:dataType :BooleanExpression .
+
+:OutsideOperatorEnumeration a samm-c:Enumeration ;
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:269]: preferredName and description missing

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+   samm:exampleValue "EA189" .
+
+:referenceToAspectModel a samm:Property ;
+   samm:preferredName "Reference to aspect model"@en ;
+   samm:description "This property describes the underlying aspect model to which this failure pattern definition should be applied to."@en ;
+   samm:characteristic :LinkToAspectModel .
+
+:booleanPropertyName a samm:Property ;
+   samm:preferredName "Boolean property name"@en ;
+   samm:description "A property name of type booleanfrom the model."@en ;
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:133]: typo booleanfrom

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl (hunk 1 above)
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: Added copyright headers

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl (hunk 2 above)
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: fixed typo

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+   samm:description "This trait ensures proper versioning of an aspect model."@en ;
+   samm-c:baseCharacteristic samm-c:Text ;
+   samm-c:constraint :RegexForAspectModelVersions .
+
+:RegexForAspectModelVersions a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "Regex for aspect model versions"@en ;
+   samm:description "This regular expression ensures that the version is [1-9].[0-9].[0.9]"@en ;
+   samm:value "^[1-9].[0-9].[0-9]$" .
+
+:StringExpressionList a samm-c:List ;
```
[the remark this answers is not in the document]
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: Deleted unused list

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+   samm:dataType :StringExpression .
+
+:DateExpressionList a samm-c:List ;
```
[the remark this answers is not in the document]
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: Deleted unused list

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl (hunk 3 above)
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: Deleted unused list

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl (hunk 5 above)
Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: fixed typo

File: io.catenax.failure_pattern/1.0.0/FailurePattern.ttl [hunk 8]
```diff
@@ -0,0 +1,278 @@
+#######################################################################
+# Copyright (c) 2022, 2024 Robert Bosch GmbH
```
Comment (agg3fe) [inline io.catenax.failure_pattern/1.0.0/FailurePattern.ttl:2]: since this is first version, only 2024 is required.

Comment (JulianSchererZF) [reply io.catenax.failure_pattern/1.0.0/FailurePattern.ttl]: Oh ja surem, fixed that as well

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**